modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/ActionsDropdown.js

```diff
@@ -88,11 +75,10 @@ function ActionsDropdown({
 
 	const isMounted = useIsMounted();
 
-	const editModeActive = !!frontendDataSetContext.itemsChanges[itemId];
+	const editModeActive = !!itemsChanges[itemId];
 	const itemChanges =
```

Comment:
Probably empty line missing. I remember the rule empty line between declaration and usage

Reply:
That's a rule in Java, currently optional in JS.